moralization/input_data.py

```diff
@@ -56,10 +56,10 @@ def get_file_type(filename: str) -> str:
         """
         Get the file extension of a given filename.
         Args:
-          filename: str:
+          filename: (str): path to the file.
```

Comment:
why is there still a colon after filename..?? please check [here](https://github.com/ssciwr/moralization-analyzer/blob/main/moralization/data_manager.py) for correct syntax